### ISSUE TITLE
Some alerts show the wrong count of new tyls

### ISSUE THREAD

**Issue by lairdshaw** [mybbgroup/Thank-you-like-system#293]
Commits dc2741b and 7a3fa32 improved alerts so that they show the count of new tyls a post has received since the last tyl alert for that post was viewed (primarily by the individual alert being clicked on in the MyAlerts modal pop-up, but alternatively by "view alerts" being clicked in that same modal pop-up).

In support of this new feature, the column `tyl_last_alerted_tyl_id` was added to the posts table, initialised for all rows to its default value of 0.

There is a problem though with this initial default value of 0: for posts which had been liked and for which alerts had been raised and viewed _prior_ to the new code introduced in those commits, the alert raised by a subsequent like will include _all_ of the previous likes in its count of "new" likes.

In other words, the value of the `tyl_last_alerted_tyl_id` column for posts which had already been liked+alerted-and-viewed should have been set to the ID of the latest tyl for each post in question rather than 0.
